build_library/grub.cfg

```diff
@@ -38,7 +41,7 @@ if [ -z "$linux_console" ]; then
 fi
 
 
-# Load a kernel and boot! $root must point at USR-A or USR-B
+# Load a kernel and boot!
 function load_coreos {
     # UEFI uses linuxefi/initrdefi instead of linux/initrd
```

Comment:
if we move the logic to select between the linux and linuxefi commands to the top level the load_coreos function could just go away.